src/pydvl/influence/frameworks/functional.py

```diff
@@ -0,0 +1,201 @@
+from functools import partial
+from typing import Callable, Dict, Generator, Iterable
+
+import torch
+from torch.func import functional_call, grad, jvp, vjp
+from torch.utils.data import DataLoader
+
+from .util import Input_type, align_structure, to_model_device
+
```

Comment:
Could you please add an `__all__` variable to the top of module to better document which functions are expected to be used outside the module?